src/models.py

```diff
@@ -0,0 +1,178 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+
+class Deshadower(nn.Module):
+    def __init__(self, in_channels, out_channels, res_blocks=9):
+        super(Deshadower, self).__init__()
+
+        # Initial conv layer
+        self.model = nn.Sequential(
+            nn.ReflectionPad2d(3),
+            nn.Conv2d(in_channels, 64, 7),
+            nn.InstanceNorm2d(64),
+            nn.ReLU(inplace=True),
+        )
+
+        # downsampling
+        in_channels = 64
```

Comment:
why in_channels are overwritten?

Reply:
It's a downsampling concern, should I just devide the initial in_channels by 2 instead?

Reply:
set default value at parameter declaration and remove that line